docs/ar/week03/03-2.md

```diff
@@ -256,56 +272,75 @@ class LeNet5(nn.Module):
 
 <!--Although `fc1` and `fc2` are fully connected layers, they can be thought of as convolutional layers whose kernels cover the entire input. Fully connected layers are used for efficiency purposes.
 -->
-على الرغم من ان fc1 و fc2 طبقات تامة الاتصال, إلا ان يمكن اعتبارهم طبقة التفافية ذات مرشح يغطي الصورة باكملها. يتم استخدام طبقات تامة الاتصال لأسباب مرتبطة بالفاعلية.
+
```

Comment:
This also looks funny. Maybe @ebetica knows how to fix it.
![image](https://user-images.githubusercontent.com/2119355/123873347-5db26100-d904-11eb-922e-d6d028d8a377.png)